kasa/cli.py

```diff
@@ -613,9 +608,7 @@ def _echo_features(
             id_: feat for id_, feat in features.items() if feat.category == category
         }
 
-    if not features:
-        return
```

Comment:
Removed this to print out all categories for consistency as some devices may not have features in all categories (i.e., hub doesn't currently have primary features).